src/schema/v2/homeView/sections/DiscoverSomethingNew.ts

```diff
@@ -0,0 +1,46 @@
+import { ContextModule } from "@artsy/cohesion"
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+import { connectionFromArray } from "graphql-relay"
+
+const marketingCollectionSlugs = [
+  "figurative-art",
+  "new-from-leading-galleries",
+  "paintings",
+  "prints",
+  "street-art",
+  "black-and-white",
+  "art-under-1000",
+  "art-for-small-spaces",
+  "cool-toned-artworks",
+]
+
+export const DiscoverSomethingNew: HomeViewSection = {
+  id: "home-view-section-discover-something-new",
+  featureFlag: "diamond_home-view-marketing-collection-categories",
+  contextModule: "" as ContextModule, // TODO: fill in with real value
+  type: HomeViewSectionTypeNames.HomeViewSectionCards,
+  component: {
+    title: "Discover Something New",
+  },
```

Comment:
**thought (non-blocking):** At the moment, both of these will get be rendered by some new `HomeViewSectionCards` component over in Eigen. But one of them will want to be rendered by a different/custom component type.

E.g. something along the lines of…

```ts
export const DiscoverSomethingNew: HomeViewSection
  id: "home-view-section-discover-something-new",

  type: HomeViewSectionTypeNames.HomeViewSectionCards,

  component: {  // ← default visual treatment
     title: "Discover Something New", 
   },

  // […]
}
```

and 


```ts
export const ExploreByCategory: HomeViewSection = {
   id: "home-view-section-explore-by-category",

   type: HomeViewSectionTypeNames.HomeViewSectionCards,

   component: {
     title: "Explore by Category",
     type: "HomeViewSectionCardChips"  // ← custom visual treatment
   }

  // […]
}
```

Is that how you're picturing this as well?

Reply:
Yeah, we'll need to pick a default visual treatment and an alternates can be have a defined `component > type` value which can be interpreted by the client/Eigen to trigger that alternate presentation.

Reply:
We can define this one as type Chips if possible bc it's the section which should be using it